src/main/resources/application.yaml

```diff
@@ -96,6 +96,8 @@ idam:
 
 oidc.issuer: ${OIDC_ISSUER_URL:https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts}
 
+environment_name: ${ENVIRONMENT_NAME:preview}
```

Comment:
please put hard coded value as PROD instead of preview, this will avoid deletion accidentally

Reply:
Addressed here: https://github.com/hmcts/rd-user-profile-api/pull/664/commits/75fa5b071dd50ece0e10d6dbc78352f4b5704a03